bgcval2/download_from_mass.py

```diff
@@ -483,7 +486,8 @@ def download_from_mass(
     if auto_download:
         shared_file_path = os.path.join(paths.shared_mass_scripts, os.path.basename(download_script_path))
         print('writing file in shared path', shared_file_path)
-        shutil.copy(download_script_path, shared_file_path)
+        try: shutil.copy(download_script_path, shared_file_path)
+        except: pass
```

Comment:
This should be something, no except pass

Reply:
well, what'd you expect here to go wrong, maybe an `IOError` I think?

Reply:
EnvironmentError?

Reply:
that's the `BaseClass` of `IOError` and such, I've not used the base class ever - you need to put whatever that should raise - am not sure what, you'd know :smiley:

Reply:
I'm not sure either - I don't make errors...


...



...


...

:laughing:

Reply:
hahaha :rofl: Seriously now - whatever happens in there when there is an exception - make the thing fail and see what exception it throws